### DISCUSSION TITLE
Problem with internet while installing command line developer tools

### DISCUSSION THREAD

**Discussion by VojtechFlidr** [Homebrew/discussions#65] [Everyday usage]
Hello guys I am completely new in the community and I have a first problem in installation Homebrew.
It is showing me this window. Some problem with network but my internet is running with no problem.
Do you know what can I do with it?
Thanks!
<img width="248" alt="Screenshot at Oct 17 01-43-35" src="https://user-images.githubusercontent.com/73003897/96322642-2f148480-101a-11eb-90a4-581c76b22b47.png">

**Comment by Rylan12** [comment]
This isn't really the right place for this as this is not a Homebrew issue.

I've found the CLT installation/upgrade process to be very finicky. I'd recommend downloading and installing the latest version of the CLT manually from the [Apple Developer downloads](https://developer.apple.com/download/more/) page. Currently, it looks like "Command Line Tools for Xcode 12" is the latest stable version.

**Reply by VojtechFlidr** [reply]
Thank you! It works
